FreeCam/MainClass.cs

```diff
@@ -20,7 +20,8 @@ class MainClass : ModBehaviour
 
 	private InputMode _storedMode;
 	private int _fov;
-	private ICommonCameraAPI _commonCameraAPI;
+    private int _nearClipPlane;
+    private ICommonCameraAPI _commonCameraAPI;
```

Comment:
Really sorry, I can't seem to see why the white space has gone funky :( 

Is it because I'm using Windows, perhaps?